zhmcclient/_lpar.py

```diff
@@ -566,6 +566,141 @@ def scsi_load(self, load_address, wwpn, lun, load_parameter=None,
             self.wait_for_status(statuses, status_timeout)
         return result
 
+    @logged_api_call
+    def scsi_dump(self, load_address, wwpn, lun, load_parameter=None,
+                  disk_partition_id=None,
+                  operating_system_specific_load_parameters=None,
+                  boot_record_logical_block_address=None,
+                  wait_for_completion=True, operation_timeout=None,
+                  status_timeout=None, allow_status_exceptions=False):
+        """
+        Load a standalone dump program from a designated SCSI device
+        in this LPAR, using the HMC operation "SCSI Dump".
+
+        This HMC operation has deferred status behavior: If the asynchronous
+        job on the HMC is complete, it takes a few seconds until the LPAR
+        status has reached the desired value. If `wait_for_completion=True`,
+        this method repeatedly checks the status of the LPAR after the HMC
+        operation has completed, and waits until the status is in the desired
+        state "operating", or if `allow_status_exceptions` was
+        set additionally in the state "exceptions".
```

Comment:
An expected status makes only sense if the LPAR is restarted after the dump program has completed. Is that what happens? Unfortunately, the HMC API book is silent about that.

Reply:
NO CHANGE.